### PR TITLE
Update config.toml

### PR DESCRIPTION
**Pull request by dontoisme**
Changing "We're Hiring" banner to recruit for user research.

### CODE REVIEW

File: site/config.toml [hunk 1]
```diff
@@ -24,8 +24,8 @@ pygmentsStyle = "manni"
 
   [params.notification]
     enable = true
-    url = "https://mattermost.com/careers"
-    text = "We're hiring!"
+    url = "https://forms.gle/z4JonkKCN8Em3zUf9"
+    text = "Complete a quick survey for the chance to receive $50 gift card"
```
Comment (dontoisme) [inline site/config.toml:28]:
```suggestion
    text = "Complete a quick survey for the chance to receive a $50 gift card"
```

**Review by dontoisme — COMMENTED** [review]
Fixing typo

**Review by cwarnermm — APPROVED** [review]
Thanks, @dontoisme!

**Comment by justinegeffen** [comment]
@cwarnermm, I am torn about whether to punctuate this or not. The previous banner ended with a "!" which was helpful. In general, CTAs don't have punctuation. What are your thoughts on this? I am 5/5 for no punctuation in CTAs, regardless of whether they're one word or a full sentence.

**Comment by justinegeffen** [comment]
Approving and merged in the meantime as this is an offline discussion. :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**